packages/dso-toolkit/src/components/badge/badge.mixins.scss

```diff
@@ -5,7 +5,8 @@
 @use "badge.variables" as badge-variables;
 
 @mixin root() {
-  border: 1px solid;
+  border-width: 1px;
+  border-style: solid;
```

Comment:
Opsplitsen in twee losse properties is eerder smaak dan noodzaak: Maar omdat kleur nu niet meer tegelijk met border-width en border-style worden gezet heb ik ervoor gekozen om dan maar helemaal van de shorthand af te stappen.